recipes/libsvm/all/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 2.8)
+project(svm C CXX)
+
+include(${CMAKE_SOURCE_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_library(svm source_subfolder/svm.cpp)
+set_target_properties(svm PROPERTIES PUBLIC_HEADER source_subfolder/svm.h)
+target_include_directories(svm PRIVATE source_subfolder)
```

Comment:
Why do we need this @uilianries? The only header is the one explicitly listed on the previous line.

Reply:
don't need, I didn't see that line.